CourseApp/Program/UserInterface.cs

```diff
@@ -0,0 +1,173 @@
+namespace CourseApp.Program
+{
+    using System;
+    using CourseApp.Program.Input;
+
+    public class UserInterface
+    {
+        public UserInterface()
+        {
+            Console.WriteLine("Hello!");
+            IsValuesEnter = false;
+        }
+
+        public bool IsValuesEnter { get; set; }
+
+        public int Year { get; set; }
+
+        public int Month { get; set; }
+
+        public int Day { get; set; }
+
+        public void StartPage()
+        {
+            var input = new InputValues();
+            Console.WriteLine(@"
+Avaible actions:
+1 - Set your birthDay
+2 - Get your age
+3 - Get date to your birthDay
+4 - Exit program
+What you want? Enter integer values.");
+            switch (input.InputInt(1, 4))
+            {
+                case 1:
+                    SetYourBirthDate();
+                    StartPage();
```

Comment:
this may lead to stackOverflow - you call function recursively multiple times

Reply:
Fixed